lib/StaticNarrative/exporter/data_exporter.py

```diff
@@ -98,12 +89,28 @@ def export_narrative_data(
     return output_data
 
 
-def _reshape_obj(obj_info: list[str | dict[str, Any]], obj_upa: str) -> list[str | dict[str, Any]]:
+def _index_obj(
+    item: dict[str, Any], type_info: dict[str, Any], indexed_data: dict[str, Any]
+) -> None:
+    """Index by object ID and save the type information."""
+    obj = item[OBJ_INFO]
+    obj_type = obj[2].split("-")[0]
+    if obj_type in IGNORED_TYPES:
+        return
+    obj_upa = generate_upa(item[OBJ_INFO])
```

Comment:
You already have `obj` defined as that above, might as well use it!

Reply:
Doh!